src/facet/inspection/_shap_projection.py

```diff
@@ -210,9 +253,9 @@ def _fit(self, shap_calculator: ShapInteractionValuesCalculator) -> None:
             cov_p_i_p_ij, var_p_i, out=cov_p_i_p_ij_over_var_p_i, where=var_p_i > 0.0
```

Comment:
See above

Reply:
If `var_p_i` is precisely 0.0, so will `cov_p_i_p_ij`, If `var_p_i` is a small positive value, then `cov_p_i_p_ij` will be a similar order of magnitude. So this division will not generate extreme results in fringe cases.